spec/example_app/db/schema.rb

```diff
@@ -23,38 +23,22 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "countries", force: :cascade do |t|
```

Comment:
As we didn't make any changes to the schema, could we drop this out, too?

In a future PR we can rerun migrations and ensure this is tidied up.